test/jit/test_typing.py

```diff
@@ -200,12 +200,14 @@ def fn2(x):
         self.checkScript(fn, [])
         self.checkScript(fn2, (torch.ones(2, 2),))
 
-        with self.assertRaisesRegex(RuntimeError, "Could not unify"):
+        with self.assertRaisesRegex(RuntimeError, "Lists must contain "
+                                    "only a single type"):
```

Comment:
Why did these error messages changes? Because of extra calls to `unifyTypes1 that weren't being made before?

Reply:
+1